From 77ce8ecc65d4e6ad0be611fdcd3720f2fb9ea34e Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sat, 3 Oct 2026 21:15:38 +0800
Subject: [PATCH 1/4] FROMLIST: ICU-23548 Mark SuperH-3 as double-conversion
 compatible

Tested good with the enclosed code snippet on SuperH-3 (QEMU user
emulation), which returned "correct" as expected.

Link: https://github.com/unicode-org/icu/pull/4255
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 icu4c/source/i18n/double-conversion-utils.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/icu4c/source/i18n/double-conversion-utils.h b/icu4c/source/i18n/double-conversion-utils.h
index 303668f9317..d1dfa54a885 100644
--- a/icu4c/source/i18n/double-conversion-utils.h
+++ b/icu4c/source/i18n/double-conversion-utils.h
@@ -154,7 +154,7 @@ int main(int argc, char** argv) {
     defined(__powerpc__) || defined(__ppc__) || defined(__ppc64__) || \
     defined(_POWER) || defined(_ARCH_PPC) || defined(_ARCH_PPC64) || \
     defined(__sparc__) || defined(__sparc) || defined(__s390__) || \
-    defined(__SH4__) || defined(__alpha__) || \
+    defined(__SH3__) || defined(__SH4__) || defined(__alpha__) || \
     defined(_MIPS_ARCH_MIPS32R2) || defined(__ARMEB__) ||\
     defined(__AARCH64EL__) || defined(__aarch64__) || defined(__AARCH64EB__) || \
     defined(__riscv) || defined(__e2k__) || \
-- 
2.56.0

